lib/components/storyblok-component.tsx

```diff
@@ -1,15 +1,25 @@
 import React from "react";
+import type { SbBlokData } from "../types";
 import { getComponent } from "../index";
 
-const StoryblokComponent = ({ blok, ...restProps }) => {
+interface StoryblokComponentProps {
+  blok: SbBlokData | false;
```

Comment:
do we want the `| false` part?

Reply:
We're returning false here:

https://github.com/storyblok/storyblok-react/blob/918a88e94202ae92a4eb9a4918a393d05d40a642/lib/index.ts#L93

Reply:
we don't!